validation-test/stdlib/Sort-ABI.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: %target-run-stdlib-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+var SortABITestSuite =  TestSuite("SortABI")
+
+SortABITestSuite.test("_merge-ABI") {
+  let result = withUnsafeTemporaryAllocation(of: Int.self, capacity: 10) {
+    buffer -> Bool in
+    _ = buffer.initialize(from: 0..<10)
+    return withUnsafeTemporaryAllocation(of: Int.self, capacity: 10) {
+      _merge(
```

Comment:
(Huh, TIL we have `-Xfrontend -disable-access-control` in the `%target-run-stdlib-swift` macro.)